server/auth/login-gov.es6

```diff
@@ -16,73 +17,60 @@ let basicAuthOptions = {
   }
 };
 
-loginGov.setup = () => {
-  console.log('------------ in loginGov.setup ', vcapServices.loginGovIdpUsername, vcapServices.loginGovIdpPassword);
-  Issuer.defaultHttpOptions = basicAuthOptions;
-  Issuer.discover('https://idp.int.login.gov/.well-known/openid-configuration')
-    .then(loginGovIssuer => {
-      console.log('----- loginGovIssuer: ', loginGovIssuer);
+const params = {
+  acr_values: 'http://idmanagement.gov/ns/assurance/loa/1',
+  nonce: `${Math.random()}-${Math.random()}`,
+  prompt: 'select_account',
+  // TODO: replace with VCAP_SERVICES
```

Comment:
In the future let's not commit stories @dubharmonic that have lingering todos in them. That would not meet the definition of done.